src/popup.html

```diff
@@ -80,6 +86,81 @@ <h4 class="i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></h4>
+                      <div class="fx-relay-segmented-control-group">
+                        <input type="radio" name="fx-relay-phone" id="fx-relay-phone-forwarding" data-forwarding="true" /> 
+                        <label class="fx-relay-label i18n-content" 
+                        id="fx-relay-phone-forwarding-button-label"
+                        data-i18n-message-id="popupPhoneMasksForwardingButton"
+                        for="fx-relay-phone-forwarding"></label>
+                      </div>
+                      <div class="fx-relay-segmented-control-group">
+                        <input type="radio" name="fx-relay-phone" id="fx-relay-phone-blocking" data-forwarding="false" /> 
+                        <label class="fx-relay-label i18n-content" 
+                        id="fx-relay-phone-blocking-button-label"
+                        data-i18n-message-id="popupPhoneMasksBlockingButton"
+                        for="fx-relay-phone-blocking"></label>
+                      </div> 
+                    </div> 
+                </div> 
+              </div>
+
+              <div class="fx-relay-phone-meta-container">
+                <p class="i18n-content fx-relay-phone-meta-description" 
+                data-i18n-message-id="popupPhoneMasksMetaDescription"></p>
+
+                <div class="fx-relay-phone-meta-details">
+                  <div class="fx-relay-phone-forward-to">
+                    <p class="i18n-content" data-i18n-message-id="popupPhoneMasksMetaForwardTo"></p>
+                    <p id="fx-relay-meta-forwarding-to">
+                      (954) 555-5555
+                    </p>
+                  </div>
+                  <div class="fx-relay-phone-registered">
+                    <p class="i18n-content" data-i18n-message-id="popupPhoneMasksMetaRegistered"></p>
+                    <p id="fx-relay-meta-registered-date">
+                      Oct 15, 2023
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+
+            <div class="fx-relay-phone-dynamic-view is-hidden">
+              <div class="fx-relay-phone-content">
+                <h1 class="i18n-content" data-i18n-message-id="popupPhoneMasksActivateYourPhoneMaskTitle"></h1>
+                <p class="i18n-content" data-i18n-message-id="popupPhoneMasksActivateYourPhoneMaskBody"></p>
+              </div>
+
+              <img class="i18n-alt-tag" data-i18n-message-id="close"  alt="" src="/images/panel-images/phone-masking/phone-mask-illustration.svg">
+              
+              <!-- TODO: Get URL for CTA -->
+              <button data-href="premium/?utm_source=fx-relay-addon&utm_medium=popup&utm_content=panel-upgrade-mask-limit-cta#pricing" data-event-action="click" data-event-label="panel-upgrade-mask-limit-cta" class="i18n-content fx-relay-c-button t-primary js-external-link" data-relay-internal="true" data-i18n-message-id="popupPhoneMasksActivateYourPhoneMaskCTA"></button>
```

Comment:
For users with the "Activate now" button, I think it should take them to `phone/`. Currently it takes them to the plan upgrade.

Reply:
Yes, this needs to be updated to point to the phone's dashboard.

Reply:
Updated - converted these to anchor tags - updated event labels as well.